docs/comparisons.md

```diff
@@ -0,0 +1,58 @@
+
+### Rollback Workflow
+Steps to rollback a deployment.
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| If using a GitOps operator like Flux, then the source manifest repository needs to be rolled back to the previous version. The best way to do so is to rollback the last configuration change commit in the Component/HLD repository. The resulting change should trigger the fabrikate pipeline to generate the last version of manifests again. | The latest changes in the Helm Release repository would need to be rolled back. The Helm Operator should then revert the changes on the next release. | N/A |
+
+### Observability
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| The greatest asset of working with the fabrikate workflow is that the GitOps source repository will reflect the exact declarative state that the cluster is expected to be in. <br><br> The [fabrikate-definitions](https://github.com/microsoft/fabrikate-definitions) repository has sets of preconfigured observability HLD components that can easily be added to any deployment. | To see the expected state of a cluster, a user must connect to a cluster and utilize either the Helm or the Kubernetes CLIs. | N/A |
+
+### Getting a Cluster Configured
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| The chosen GitOps operator needs to be added to the cluster and configured to the final generated manifest repository. <br><br> The [bedrock](https://github.com/microsoft/bedrock) and [bedrock-cli](https://github.com/microsoft/bedrock-cli) projects can be leveraged to configure clusters as well as supporting pipelines in an automated and repeatable pattern. | The Flux operator and Helm Operator must be installed onto a cluster then configured to the appointed Helm Release repository.| N/A |
+
+### Helm version support
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| Both Helm 2 and 3 are supported. <br><br> **Note** The latest releases of fabrikate support Helm v3. To use fabrikate with Helm2, either use an older release (v__ and older) or build a binary off the [helm2](https://github.com/microsoft/fabrikate/tree/helm2) branch. | Both Helm 2 and 3 are supported | N/A |
```

Comment:
Helm vs Fab versions outlined here: https://github.com/microsoft/fabrikate/wiki/Roadmap-Updates#release-v1-and-v2